### DISCUSSION TITLE
Improved quality and bitrate for JioFiber with JioSTB subscription

### DISCUSSION THREAD

**Discussion by vaibhavramdasi** [rabilrbl/jiotv_go#408] [Q&A]
Hi,
I have a jio fiber connection with JioSTB subscription. I observed that we are fetching channels available using getMobileChannels api endpoint. Will it be possible to get higher bitrate channel endpoints if JioSTB subscription is there? CUrrently, max resolution is limited to 720p only.

**Comment by rabilrbl** [comment]
Not possible
